<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cubo</class>
 <widget class="QDialog" name="Cubo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cubo</string>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>410</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>415</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Escalar Cubo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>470</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Escalar</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="escalaBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>470</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Dibujar Cubo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Angulo de Rotacion</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="BoxX">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-400</number>
   </property>
   <property name="maximum">
    <number>400</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>420</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Eje X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>470</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="BoxY">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>410</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-300</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ValorAngulo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>15.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>460</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Eje Y</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>611</width>
     <height>351</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>415</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>390</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Reflejar Cubo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Trasladar Cubo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>575</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="escalaBox_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>630</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>610</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Angulo de Rotacion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>610</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Escalar Polygono</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ValorAngulo_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>630</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>15.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>550</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Trasladar Polygono</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>630</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Escalar</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="BoxX_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>570</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-400</number>
   </property>
   <property name="maximum">
    <number>400</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>560</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Dibujar Cubo</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="BoxY_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>570</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-300</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>570</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>580</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Eje X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>620</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Eje Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>550</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Reflejar Poligono</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>575</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>630</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
